### PR TITLE
[FLOSS H3] links: Replace tons of http with https

### PR DESCRIPTION
**Pull request by Eiskasten**
## Basics

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation (see [Documentation Guidelines](https://www.libelektra.org/devgettingstarted/documentation))
- [ ] I fixed all affected decisions (see [Decision Process](https://www.libelektra.org/decisions/decision-process))
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://www.libelektra.org/devgettingstarted/coding))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.

### CODE REVIEW

**Comment by markus2330** [comment]
Thank you for the PR!

Does this replace all http:// links? Which ones are missing?

**Review by Bujuhu — CHANGES_REQUESTED** [review]
Remember to add your changes to docs/news/_prepare_next_release.md

File: doc/WHO.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ configuration values can be shared (integrated).
 Known users:
 
 - [Oyranos](https://oyranos.org)
-- [LCDproc](http://lcdproc.omnipotent.net/) (in progress)
```
[the remark this answers is not in the document]
Comment (Eiskasten) [reply doc/WHO.md:14]: You are right, the cert is only valid for different subdomains.

**Comment by Eiskasten** [comment]
> Thank you for the PR!
> 
> Does this replace all http:// links? Which ones are missing?

Not all links are replaced. There are three main reasons why not all links can be replaced:

- there is no https link available
- the http link is part of a xml namespace, https links are per definition another namespace
- the link is part of a license or included library which may result in an incorrect license detection

**Comment by flo91** [comment]
@Eiskasten 
Please rebase and resolve the merge-conflict in the file `src/tools/website/README.md`.

**Comment by markus2330** [comment]
@Eiskasten please rebase

**Comment by markus2330** [comment]
@Eiskasten not all man pages were regenerated.

**Comment by markus2330** [comment]
I think you need to keep the http link for lcdproc.omnipotent.net, their https setup seems to be broken.

**Comment by markus2330** [comment]
Perfect, all is well now!

**Pull request merged**